test/second_q/problems/test_vibrational_structure_problem.py

```diff
@@ -55,7 +41,7 @@ def test_second_q_ops_without_transformers(self):
         """Tests that the list of second quantized operators is created if no transformers
         provided."""
         expected_num_of_sec_quant_ops = 4
-        expected_len_of_vibrational_op = 47
+        expected_len_of_vibrational_op = 46
```

Comment:
In case somebody comes across this change: that is because the all-identity with coefficient `0` was dropped :+1: